LiverSegments/Logic/vtkLiverSegmentsLogic.cxx

```diff
@@ -89,7 +89,7 @@ int vtkLiverSegmentsLogic::SegmentClassificationProcessing(vtkMRMLModelNode *cen
     vtkSmartPointer<vtkIntArray> centerlineSegmentIDs = vtkIntArray::SafeDownCast(pointData->GetScalars());
 
     if(centerlineSegmentIDs == nullptr) {
-        std::cout << "No PointData in Model" << std::endl;
+        std::cout << "Error: No PointData in centerline model" << std::endl;
         return -1;
```

Comment:
One thing that I just realized. The return values, in Slicer, generally are 0 for failure and >0 for success. I suppose the reason behind it is that you can do direct interrogations (e.g., `if (this->ComputeCenterlines())` as opposed to `if (this->ComputeCenterlines() == 0)`)